src/devcenter/azext_devcenter/aaz/latest/devcenter/admin/network_connection/_update.py

```diff
@@ -74,15 +78,12 @@ def _build_arguments_schema(cls, *args, **kwargs):
         _args_schema.domain_name = AAZStrArg(
             options=["--domain-name"],
             arg_group="Properties",
-            help="Active Directory domain name.",
+            help="Active Directory domain name",
         )
         _args_schema.domain_password = AAZStrArg(
             options=["--domain-password"],
             arg_group="Properties",
-            help="The password for the account used to join domain.",
-            blank=AAZPromptInput(
-                msg="Domain password:",
-            ),
```

Comment:
Why removed the prompt input support?

Reply:
Unintended, re-added this and also updated the aaz PR